docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,138 @@
+-   Tap on a trigger block to select all the trigger points within the block. You will see selection 
+    handles appear.  Tap again on a selected trigger block to step through the
+    trigger points within the block. 
+
+![Timeline trigger handles](/docs/images/Timeline-Handles.png)
+
+- Tap on more triggers to add them to the selection.
+
+- Draw a selection marquee box over the trigger points you want to select. Only the trigger points enclosed by the selection box will be selected, so if you want to select all points in a block they all need to be inside the box. This is a quick way to select one trigger point without having to "multi-tap" on a block.
+
+- To clear all selection use the context menu button \{Clear Trigger Selection\} or tap in an empty area of the Timeline View.
+
+### Using arrow buttons to move trigger selection
+
+The left/right arrow buttons for the track will move your selection to the previous trigger point or the next trigger point.
+
+- Left arrow - moves selection to the previous trigger point on the track.
+- Right arrow - moves selection to the next trigger point on the track.
+
+If nothing is selected, the right arrow will select the first trigger on the track and the left arrow will select the last.
+
+### Using the wheels to edit triggers
+
+When you have one or more trigger points selected, the wheels control the time, level and fade of those triggers (some types of trigger do not have all these parameters).
+
+- Wheel A sets the time.
+- Wheel B sets the level.
+- Wheel C sets the fade.
+
+You can change multiple triggers at the same time by selecting them all and using the wheels.
+
+### Using the overview bar
+
+The **Overview Bar** at the bottom of the Timeline View gives you a miniature view of all the events in the track.
+
+![Timeline overview bar](/docs/images/Timeline-Overview-Bar.png)
+
+- The handles at the left and right of the Overview Bar set the start and end points of the main view above. This gives you an easy way to zoom in on a particular area.
+
+- When you are zoomed in you can drag on the zoomed area in the bar to move the main view (the same as using the Hand tool and dragging on the main view).
+
+- The edit cursor is shown in grey.
+
+- The current live time is shown in blue.
+
+### Deleting triggers
+
+You can delete trigger blocks by pressing \<Delete\> then tapping on the block you want to delete, or drawing a marquee selection box over the triggers to be deleted.
+
+You can also delete tracks or delete rows from tracks using this method.
+
+### Setting track legends and halo colours on tracks
+
+To help you identify different tracks, you can set halo colours. This colour outlines the track and also sets the colours of the trigger blocks which makes them easier to see in the overview bar.
+In addition you can set different colours for the playback actions within each track as shown in Track 2 in the picture below.
```

Comment:
I'm not sure what `playback actions` refers to here... I don't think `action` is a term that we've used elsewhere?

Reply:
I mean the row in the track that refers to a particular playback. Would it be better to say "you can set different colours for each playback row within the track"?

Reply:
Ah, I see what you mean. What's actually happening here is that you're setting the halo of the playback itself - you can test this by setting the halo colour of a playback and then assigning it into the timeline. The playback's halo colour gets shown in the timeline.

Reply:
Right, didn't spot that, will update